index.php

```diff
@@ -231,7 +231,7 @@ function get_http_security () {
 				<form id="search_form" method="get" onsubmit="return doSearch()">
             <div class="seven columns alpha">
 			<div id="header_logo" title="To search, enter some search terms, then click a provider." onclick="if ( $('#query').val() ) { doSearch(); }">
-				<img src="cc-search.png" alt="CC Search" />
+				<img src="cc-search.png" alt="CC Search Portal" />
```

Comment:
Since this PR isn't focused on changing the image, the alt text here should remain unchanged and still be `CC Search`. The alt text has to match what the graphic has within it to be adequately accessible.